installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Lets standardize on naming convention. Use all CAPS for objects and their keys. For keys within a node object, lets remove redundancy. For example "API_BUILD" : {	"API_BUILD_URI": "", "API_BUILD_URI_DEV": "" }

lets change it to "API" : {	"BUILD_URI": "", "BUILD_URI_DEV": "" }

Please take care of this in every place.